src/panel/common/commonActions.js

```diff
@@ -66,9 +70,11 @@ function isInvalidTitle(title) {
 	}
 }
 
-function isInvalidTags(tagArr) {
-	for (let i=0; i<tagArr.length; i++) {
-		if (tagArr[i] === "") {
+
+function isInvalidTags(tokenArr) {
+	for (let i=1; i<tokenArr.length; i++) {
+		// Tags are invalid if there are two consecutive operators
+		if (tokenArr[i-1]["tokenType"] === "BINARY_OP" && tokenArr[i]["tokenType"] === "BINARY_OP") {
```

Comment:
Perhaps one way to check validity would be to step through the `tokenArr`, keeping track of the number of literals on the (imaginary) stack, and checking that at no point does an operator attempt to pop elements out of an empty stack. And at the end checking that there is only one element left on the stack.